src/integrations/connectors/snowflake/snowpach/main.go

```diff
@@ -145,7 +145,7 @@ func connect() (*sqlx.DB, error) {
 }
 
 func read(db *sqlx.DB) error {
-	stage := env("PPS_PIPELINE_NAME", true) + env("PACH_JOB_ID", false)
+	stage := env("PPS_PROJECT_NAME", false) + "_" + env("PPS_PIPELINE_NAME", true) + "_" + env("PACH_JOB_ID", false)
```

Comment:
Sure — this was just based on current code.

Reply:
I switched it over to import the environment variable names from package `client`, and also extracted it into a function since `writeTable` does the same thing one function down.